pxr/imaging/hd/camera.h

```diff
@@ -74,6 +74,11 @@ PXR_NAMESPACE_OPEN_SCOPE
     (shutterOpen)                                                 \
     (shutterClose)                                                \
     (exposure)                                                    \
+    (exposureTime)                                                \
+    (exposureIso)                                                 \
+    (exposureFStop)                                               \
+    (exposureResponsivity)                                        \
+    (exposureCompensation)                                        \
```

Comment:
These new params are missing from the hydra camera schema: https://github.com/PixarAnimationStudios/OpenUSD/blob/release/pxr/imaging/hd/cameraSchema.h ... as well as the two modes of scene index emulation:
https://github.com/PixarAnimationStudios/OpenUSD/blob/release/pxr/imaging/hd/dataSourceLegacyPrim.cpp#L1036
... and ...
https://github.com/PixarAnimationStudios/OpenUSD/blob/release/pxr/imaging/hd/sceneIndexAdapterSceneDelegate.cpp#L1237

... although it looks like for those, since you aren't doing any weird type conversion, the fallbacks might be ok and if you just add the names to dataSourceLegacyPrim.cpp that'll be enough.

Reply:
Hi @tcauchois - since I see you've already implemented some of the other related stuff, I'm guessing you've already made the changes to hd/hdSchemaDefs.py that would address this?  If not, I'm happy to push these fixes myself.

One note - while testing this, I noticed that `hdGenSchema.py` requires `jinja2 >= 3.0` - I'll make a separate PR updating that in `VERSIONS.md`.

Reply:
Here's the separate PR to update the jinja2 req:

https://github.com/PixarAnimationStudios/OpenUSD/pull/3314

Reply:
I've made the schemaDefs changes already; I think modulo the discussions downthread about ComputeLinearExposureScale and changing the transport tokens, this is almost ready to go.  Thanks for the VERSIONS.md update!

Reply:
Hi @tcauchois just to be really clear here: are you saying the only update to this PR you require from us is the rework of the exposure scale calculation to be moved to a separate function on HdCamera that gives the (linear) value from UsdGeomCamera::ComputeExposureScale(), and return HdCamera::GetExposure() to its original behaviour?

Reply:
I've pulled this down and added the Hydra 2 functionality and started reconciling issues with closed source code.  My plan was to take the PR with my additions, plus the changes mentioned (ComputeExposureScale and the different tokens), if that's ok with you, in which case there's no further action required on your part.  If you'd like me to upload my changes as a new PR instead, for you all to look at or iterate on, let me know.

Reply:
Would be great if you could just so we can all see what’s actually going in